static/src/javascripts/bootstraps/commercial-hosted-legacy.ts

```diff
@@ -1,71 +1,19 @@
 /**
- * This file is deprecated. Only used for “Hosted” pages
- * All other pages use the standalone bundle.
+ * This file is deprecated. Only used for legacy “Hosted” pages
```

Comment:
Good question. I think we consider 'hosted pages' deprecated in the sense we aren't adding features, they are very rarely used but we are expected to do bug fixes.

I'll reword the comment for now as it's confusing.

Reply:
Thanks. I think this will also be the case for the files in https://github.com/guardian/frontend/tree/main/static/src/javascripts/projects/commercial/modules/hosted

Reply:
I think I'll leave the `modules/hosted` files as they are as I intend to move them over to the commercial repo and bundle there. At which point we can fully delete everything under commercial.